.vscode/extensions.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
add the ones from the readme here, update the readme section to point to this file